et_converter/pytorch2chakra_converter.py

```diff
@@ -554,62 +466,6 @@ def get_chakra_node_type_from_pytorch_node(self, pytorch_node: PyTorchNode) -> i
             return COMP_NODE
         return INVALID_NODE
 
-    def identify_data_dependency(self) -> None:
-        """
-        Identifies data dependencies between nodes using tensor input/output
-        relationships.
-
-        Determines the relationships based on whether the tensors use storage IDs
-        or tensor IDs.
-        """
-        self.logger.info("Identifying data dependencies among nodes.")
-        self.identify_data_dependency_with_storage_id()
-        self.identify_data_dependency_with_tensor_id()
-
-    def identify_data_dependency_with_storage_id(self) -> None:
```

Comment:
The key role of the et_converter is to convert control dependencies into data dependencies. I've realized we don't need storage IDs or tensor IDs for that. What we discussed in the last meeting was enforcing dependencies from GPU ops to CPU ops. I believe this can be achieved with [the Lightweight Critical Path Analysis](https://hta.readthedocs.io/en/latest/source/features/lightweight_critical_path_analysis.html).

Reply:
Got it! I will take a look.